pkg/kapp/cmd/warning_flags.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	cmdcore "github.com/k14s/kapp/pkg/kapp/cmd/core"
+	"github.com/spf13/cobra"
+)
+
+type WarningFlags struct {
```

Comment:
what do you think about moving this flag into KubeAPIFlags (https://github.com/vmware-tanzu/carvel-kapp/blob/develop/pkg/kapp/cmd/core/kube_api_flags.go). they all seem to be related to control kapp's interaction with kube api.

Reply:
That does sound reasonable. I was thinking of using this as an extension to include more flags related to warnings in the near future(if necessary), for example deduplication, warnings as error, etc,. We can definitely use kube api flags for the new flags as well.
Let me know which one sounds better.

Reply:
> for example deduplication

i dont think this needs a separate flag. we can just do it by default?

> warnings as error

that's an interesting idea.

Reply:
this leads me to thinking should we actually call this --kube-api-warnings since these warnings are coming from kube api and not general warnings from kapp?

Reply:
> i dont think this needs a separate flag. we can just do it by default? 

Yes, even I haven't seen a separate flag for deduplication anywhere.

> that's an interesting idea.

This is also provided by kube APIs.

> this leads me to thinking should we actually call this --kube-api-warnings since these warnings are coming from kube api and not general warnings from kapp?

I am just worried that the names would be too long then, for warnings as error, it would be something like --kube-api-warnings-as-error. However if we continue with --warnings, then we should definitely show/hide warnings produced by kapp as well (if there are any).

Reply:
Based on discussion, continuing with --warning. In future kapp's own warning will be included under this.